infra-library.sh

```diff
@@ -137,5 +137,11 @@ function create_gke_test_cluster() {
   local -n _custom_flags=$1
   local -n _test_command=$2
 
-  run_kntest kubetest2 gke "${_custom_flags[@]}" --test-command="${_test_command[*]}"
+# We are disabling logs and metrics on Boskos Clusters by default as they are not used. Manually set GKE_DISABLE_TELEMETRY to false to enable logging
+  if [ "$GKE_DISABLE_TELEMETRY" == "false" ]
```

Comment:
Please keep the Bash style consistent across the scripts to change it to `if [[ "$GKE_DISABLE_TELEMETRY" == "false" ]]; then`.

Also it's more natural to set default value as false, so I would recommend something like `"${GKE_ENABLE_TELEMETRY:=false}"`

Reply:
fixed